File: Backend/app/Models/Reservation.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsTo;


class Reservation extends Model
{
    use HasFactory;
    public $timestamps = false;

    protected $fillable = [
        'number_of_guests',
        'checkin_date',
        'checkout_date',
        'name',
        'phone',
        'user_id',
        'desk_id',
        'closed',
        'comment'
    ];


    public function desk():BelongsTo
    {
        return $this->belongsTo(Desk::class);
    }

    public function user():BelongsTo
    {
        return $this->belongsTo(User::class)->whereNotNull('id');
    }
}

File: MG_Admin_GUI_v2.2/Models/reservation.cs

namespace MG_Admin_GUI.Models;

public partial class reservation
{
    public ulong id { get; set; }

    public int number_of_guests { get; set; }

    public DateTime checkin_date { get; set; }

    public DateTime checkout_date { get; set; }

    public string name { get; set; } = null!;

    public string phone { get; set; } = null!;

    public ulong desk_id { get; set; }

    public ulong? user_id { get; set; }

    public bool closed { get; set; }

    public string? comment { get; set; }

    public DateTime? deleted_at { get; set; }

    public virtual desk desk { get; set; } = null!;

    public virtual user? user { get; set; }
}
